diff --git a/dist/Page.d.ts b/dist/Page.d.ts
index 158d677ffb485fef91b8b5785377f067cce42845..05772ab1736625d547472fa0f6f9515a584e95fa 100644
--- a/dist/Page.d.ts
+++ b/dist/Page.d.ts
@@ -9,6 +9,8 @@ import '../assets/Page.scss';
 export type PageProps = (StandardProps & HTMLProps<HTMLDivElement> & GenericHeaderProps & Omit<ServiceNavigationProps, 'items'> & Omit<FooterProps, 'navigation'> & Partial<PhaseBannerProps> & {
     /** Location for the Back link */
     backHref?: string;
+    /** Text for the Back link */
+    backText?: string;
     /** List of links to parent pages */
     breadcrumbs?: Breadcrumb[];
     /** The content that displays in the page. */
diff --git a/dist/Page.js b/dist/Page.js
index f015a7dae2b9fd2db0695d438c00cea324dd2059..e3f67873a7f8dc8906ca03226e15c5a51a55c50e 100644
--- a/dist/Page.js
+++ b/dist/Page.js
@@ -14,7 +14,7 @@ import { ServiceNavigation } from '@not-govuk/service-navigation';
 import { SkipLink } from '@not-govuk/skip-link';
 import { WidthContainer } from '@not-govuk/width-container';
 import '../assets/Page.scss';
-export const Page = ({ backHref, breadcrumbs, children, classBlock, classModifiers: _classModifiers, className: _className, department, feedbackHref, footerContent, footerNavigation, govUK, logo, maxContentsWidth, meta, metaTitle, navigation, organisationHref, organisationText, phase, phaseBannerContent, serviceHref, serviceName, signOutHref, signOutText, ...attrs }) => {
+export const Page = ({ backHref, backText, breadcrumbs, children, classBlock, classModifiers: _classModifiers, className: _className, department, feedbackHref, footerContent, footerNavigation, govUK, logo, maxContentsWidth, meta, metaTitle, navigation, organisationHref, organisationText, phase, phaseBannerContent, serviceHref, serviceName, signOutHref, signOutText, ...attrs }) => {
     const classModifiers = [...(Array.isArray(_classModifiers)
             ? _classModifiers
             : [_classModifiers]), department];
@@ -50,6 +50,6 @@ export const Page = ({ backHref, breadcrumbs, children, classBlock, classModifie
                                     ? (_jsx(A, { href: feedbackHref, children: "feedback" }))
                                     : 'feedback', " will help us to improve it."] })) }))] }), _jsx("div", { className: classes('middle'), children: _jsxs(WidthContainer, { maxWidth: maxContentsWidth, className: classes('container'), children: [breadcrumbs?.length
                             ? (_jsx(Breadcrumbs, { id: "breadcrumbs", items: breadcrumbs }))
-                            : (!backHref ? null : (_jsx(BackLink, { id: "back-link", href: backHref }))), _jsx("main", { id: mainId, className: classes('main'), children: children })] }) }), _jsx("footer", { className: classes('footer'), children: _jsx(Footer, { ...footerProps, children: footerContent }) })] }));
+                            : (!backHref ? null : (_jsx(BackLink, { id: "back-link", href: backHref, text: backText }))), _jsx("main", { id: mainId, className: classes('main'), children: children })] }) }), _jsx("footer", { className: classes('footer'), children: _jsx(Footer, { ...footerProps, children: footerContent }) })] }));
 };
 export default Page;
